dnYara/CompiledRules.cs

```diff
@@ -31,8 +41,7 @@ public CompiledRules(string filename)
 
         public YR_RULES GetStruct()
```

Comment:
Useless, only ptr is used. Can be removed

Reply:
Removed in latest.